website/static/js/filepage/index.js

```diff
@@ -218,6 +196,27 @@ var FileViewPage = {
             ]);
         }
 
+        var editPane = function() {
+            if (ctrl.editor) {
+                return m('.btn' + (ctrl.editor.selected ? '.btn-primary' : '.btn-default'), {
+                    onclick: function (e) {
+                        e.preventDefault();
+                        // atleast one button must remain enabled.
+                        if ((!ctrl.editor.selected || panelsShown > 1)) {
+                            ctrl.editor.selected = !ctrl.editor.selected;
+                            ctrl.revisions.selected = false;
+                        }
+                    }
+                }, ctrl.editor.title);
+            }
+        };
+        var editDisplay = function() {
+            ctrl.triggerResize();
+            if (ctrl.editor && !ctrl.editor.selected) {
```

Comment:
This can be refactored along the lines of:

```js
var display = (ctrl.editor && ctrl.editor.selected) ? 'block' : 'none';
return m('.col-sm-12', {style: {display: display}, ctrl.editor);
```